tests/python/pytorch/graphbolt/test_subgraph_sampler.py

```diff
@@ -0,0 +1,25 @@
+import dgl
+import dgl.graphbolt
+import gb_test_utils
+import torch
+import torchdata.datapipes as dp
+
+
+def test_SubgraphSampler():
+    graph = gb_test_utils.rand_csc_graph(20, 0.15)
+
+    def sampler_func(data):
+        adjs = []
+        seeds = data
+
+        for hop in range(2):
+            sg = graph.sample_neighbors(seeds, torch.LongTensor([2]))
+            seeds = sg.indices
+            adjs.insert(0, sg)
+        return seeds, data, adjs
+
+    minibatch_dp = dp.map.SequenceWrapper(torch.arange(10)).batch(2)
```

Comment:
why not use existing `graphbolt.MinibatchSampler`?

Reply:
Added